Assets/Scripts/ssjAug2018/DeilveryTarget/DeliveryTarget.cs

```diff
@@ -0,0 +1,54 @@
+﻿using pdxpartyparrot.Core.Actors;
+
+using UnityEngine;
+
+// TODO: Lots. Basic class implementation so I can reference this in the manager
+namespace pdxpartyparrot.ssjAug2018.DeliveryTargets
+{
+    [RequireComponent(typeof(Collider))]
+    [RequireComponent(typeof(AudioSource))]
```

Comment:
No need for an AudioSource (any audio we'd do here would be one shot audio rather than positional)